doc/CODE_OF_CONDUCT_BASE.md

```diff
@@ -0,0 +1,40 @@
+The OpenJS Foundation and its member projects use [Contributor Covenant v2.0](https://contributor-covenant.org/version/2/0/code_of_conduct) as their code of conduct. The full text is included [below](#contributor-covenant-code-of-conduct-v20) in English, and [translations](https://www.contributor-covenant.org/translations) are available on the Contributor Covenant website.
+
+## Commitment
+
+All recipients of reports commit to:
+
+- maintain the confidentiality with regard to the reporter and victim of an incident, and
```

Comment:
Should there be any case where there are exceptions for that? Aka, the reporter themselves are violating the CoC in their report?

Reply:
CODE_OF_CONDUCT_BASE.md is taken all from the CODE_OF_CONDUCT.md file from this repository. I just got the first part and moved it into a BASE.md suffix file, for the `patch-my-code-of-conduct` to work.

Reply:
Yeah, let's focus just on infra here.